installation/proc-install-rhdh-ocp-operator.adoc

```diff
@@ -60,7 +41,7 @@ metadata:
 data:
   app-config-rhdh.yaml: |
     app:
-      title: Red Hat Developer Hub
+      title: {product}
```

Comment:
The `{product}` doesn't work in yaml block.
```suggestion
      title: Red Hat Developer Hub
```

Reply:
Updated `[source]` so that the attribute values show up in the code bloc output

Reply:
and it worked in Pantheon too? :smile:

Reply:
We won't know until we merge! :) We can only stage merged changes in Pantheon.